src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/ConnectionTest.php

```diff
@@ -520,6 +536,20 @@ public function testItRetriesTheMessage()
         $connection->publish('{}', [], 5000, $amqpStamp);
     }
 
+    public function testItRetriesTheMessageWithDailyDelayQueues()
+    {
+        $delayExchange = $this->createMock(\AMQPExchange::class);
+        $date = (new \DateTimeImmutable())->format('Y-m-d');
+        $delayExchange->expects($this->once())
+            ->method('publish')
+            ->with('{}', "delay_messages__5000_retry_$date", \AMQP_NOPARAM);
+        $connection = $this->createDelayOrRetryConnection($delayExchange, '', "delay_messages__5000_retry_$date", true);
+
+        $amqpEnvelope = $this->createMock(\AMQPEnvelope::class);
+        $amqpStamp = AmqpStamp::createFromAmqpEnvelope($amqpEnvelope, null, '');
```

Comment:
```suggestion
        $amqpStamp = AmqpStamp::createFromAmqpEnvelope($this->createMock(\AMQPEnvelope::class), null, '');
```
No need for one time variable